lib/archethic_web/graphql_schema.ex

```diff
@@ -97,6 +97,20 @@ defmodule ArchEthicWeb.GraphQLSchema do
         {:ok, Resolver.nodes()}
       end)
     end
+
+    @desc """
+    Query the network to list the transaction on the type
+    """
+    field :network_transactions, list_of(:transaction) do
+      arg(:type, :string)
+      arg(:page, :integer)
+
+      resolve(fn args, _ ->
+        type = Map.get(args, :type)
+        page = Map.get(args, :page, 1)
+        {:ok, Resolver.network_transactions(String.to_atom(type), page)}
```

Comment:
You should avoid to use `String.to_atom` as it may create a overflow of the atom list in the Erlang VM. It's a security issue, and a way to bring down nodes for attackers.

But you can either create a function to parse stringified type to atomized type, or use `String.to_existing_atom`